package.json

```diff
@@ -8,22 +8,38 @@
   "license": "Apache-2.0",
   "scripts": {
     "compile": "tsc",
-    "lint": "tslint --project .",
     "test": "ava",
     "clean": "gts clean",
-    "fix": "gts fix"
+    "cli": "node ./build/src/cli/index.js",
```

Comment:
you might consider having a separate `package.json`/project for the CLI.

Reply:
I think we may want to set it up somehow to add the final index.js into the `bin` directory to run similar to `ava` and `gts`.

Reply:
Will look into this